src/main/scala/dpla/ingestion3/harvesters/api/MdlHarvester.scala

```diff
@@ -79,58 +77,31 @@ class MdlHarvester(shortName: String,
     * @param start Uses start as an offset to paginate.
     * @return ApiSource or ApiError
     */
-  private def getSinglePage(start: String): ApiResponse =
-    getUri(queryParams.updated("start", start)) match {
-      // Error building URL
+  private def getSinglePage(start: String): ApiResponse = {
```

Comment:
This was changed because I couldn't see how building a URL could fail so this was an unnecessary check.